browser/BUILD.gn

```diff
@@ -381,9 +381,15 @@ source_set("browser_process") {
       ]
     } else {
       sources += [
+        "brave_rewards/checkout_dialog.cc",
+        "brave_rewards/checkout_dialog.h",
         "brave_rewards/tip_dialog.cc",
         "brave_rewards/tip_dialog.h",
       ]
+
+      deps += [
+        "//third_party/libaddressinput"
```

Comment:
this dep isn't used in checkout_dialog, but there are other deps that are and they aren't all listed here (components/payments/content, components/payments/core, etc...)

Reply:
resolved, deps updated: https://github.com/brave/brave-core/pull/7557/files#diff-1316014f48cc38322b1104f00c2c513d6aadabf847c700579d0905a586c9a608R395